RecoTracker/IterativeTracking/python/DetachedTripletStep_cff.py

```diff
@@ -252,15 +254,19 @@
 detachedTripletStep = ClassifierMerger.clone()
 detachedTripletStep.inputClassifiers=['detachedTripletStepClassifier1','detachedTripletStepClassifier2']
 
+trackingPhase1.toReplaceWith(detachedTripletStep, TrackMVAClassifierDetached.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorDetachedTripletStep_Phase1'),
```

Comment:
the pre-DNN version was cloning from detachedTripletStepClassifier1. It may be worth to revert (to also be in sync with the highBetaStar and pp_on_AA).

Reply:
Reverted back to clone from detachedTripletStepClassifier1.